frame/contracts/src/weights.rs

```diff
@@ -208,9 +208,9 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: Contracts PristineCode (r:0 w:1)
 	// Storage: Contracts OwnerInfoOf (r:0 w:1)
 	fn instantiate_with_code(c: u32, s: u32, ) -> Weight {
-		(283_144_000 as Weight)
+		(379_283_000 as Weight)
 			// Standard Error: 0
-			.saturating_add((140_000 as Weight).saturating_mul(c as Weight))
+			.saturating_add((128_000 as Weight).saturating_mul(c as Weight))
 			// Standard Error: 0
 			.saturating_add((2_000 as Weight).saturating_mul(s as Weight))
 			.saturating_add(T::DbWeight::get().reads(5 as Weight))
```

Comment:
This is expected. The instrumentation cost goes down. However, the overall cost goes up. No idea where this comes from.